src/main/java/com/garbagemule/MobArena/waves/MACreature.java

```diff
@@ -131,29 +131,51 @@ public LivingEntity spawn(Arena arena, World world, Location loc) {
             case "magmacubehuge":
                 ((Slime) e).setSize(4);
                 break;
+            case "babydrowned":
+            case "babyhusk":
             case "babyzombievillager":
             case "babyzombie":
                 ((Zombie) e).setBaby(true);
                 break;
+            case "babyhoglin":
+                ((Hoglin) e).setImmuneToZombification(true);
+                ((Hoglin) e).setBaby();
+                break;
+            case "babypiglin":
+                ((PiglinAbstract)e).setBaby();
```

Comment:
Add a space between the first `)` and the `e` here for consistency.

Reply:
Didn't see that one, yikes!